openmc/cell.py

```diff
@@ -563,7 +563,7 @@ def create_surface_elements(node, element):
                 if isinstance(node, Halfspace):
                     path = "./surface[@id='{}']".format(node.surface.id)
                     if xml_element.find(path) is None:
-                        xml_element.append(node.surface.create_xml_subelement())
+                        xml_element.append(node.surface.to_xml_element())
```

Comment:
Yes, I wanted to do it with Cell, Lattice, and Universe too but they are a bit more complicated so I held off for now.